torch/fx/symbolic_trace.py

```diff
@@ -206,6 +206,10 @@ def forward(*args, **kwargs):
             torch.nn.Module.__getattr__ = orig_getattr  # type: ignore
         return self.graph
 
+    def _proxy_placeholder(self, name: str, type_expr: Optional[Any] = None) -> Proxy:
+        return Proxy(self.create_node('placeholder', name, (), {}, type_expr=type_expr), self)
```

Comment:
I don't think this is used anywhere. This and the import above should be deleted